Some styling changes

elsigh opened the pull request:
Keep the home-link-logo top left.
Give build name more room on releases page.
Changes default to last 10 releases.
Adds github link/logo to footer.
Only show the video promo and text on localhost or github.
(maybe that should only be github?)
A few little spacing additions.

bslatkin commented:
Thanks for making tweaks! Some comments below. Probably need to rebase this too.

I'm gonna be annoyingly nitpicky. I really wanted to keep the logo in top right so the top-level title text is uniform and does not move throughout the whole UI. Do you feel it's unusable on the right? I spent quite a bit of time to get it to wrap correctly on mobile. If you resize the window you'll see what I mean.

Instead of modifying frontend.py to look at the hostname I think adding something to config.py that defaults to `False` sounds better to me.

The `GitHub-Mark-32px.png` file is missing from the patch.

I changed the video to <https://www.youtube.com/watch?v=1wHr-O6gEfc> instead.

Any reason you removed the `ellipsis-overflow` class on that one div?

elsigh commented:
OK, cool - rebased!

It's great to be picky, I didn't know if you'd want this change at all ;0

I feel like as it is with this change the top level title text is always in the same place - just right of the Home link / depicted logo. I do feel like it's unusable on the right - it's an uncommon place for a home link and on a wide screen it's a large drag to get there (and I kinda think that the desktop use case for dpxdt is prime). Now there's no need to wrap differently on mobile is there? Do you think build names tend to be really long? For me they're fairly short (Production, Staging, Testing, etc..) but as it stands the build name will overflow ellipsis and always be on top. Maybe I should deploy this change as a test version to the demo appengine app (where there's lots of filled in stuff) and we can see it in action?

Changed SHOW_VIDEO_AND_PROMO_TEXT to come from config.

Added GitHub-Mark-32px.png back.

The removal of ellipsis-overflow is unnecessary, added that back in.
